build/SharedFx.targets

```diff
@@ -352,19 +352,10 @@
 
   <Target Name="PackSharedFx">
     <ItemGroup>
-      <CrossGenOutput Include="$(SharedFxCrossGenDirectory)**\*.dll" />
-      <CrossGenSymbols Include="$(SharedFxCrossGenDirectory)**\*" Exclude="@(CrossGenOutput)" />
+      <CrossGenOutput Include="$(SharedFxCrossGenDirectory)**\*" />
     </ItemGroup>
 
-    <!-- Create symbols archive -->
-    <ZipArchive
-      File="$(SharedFxOutputPath)$(SharedFxInstallerName)-symbols-$(PackageVersion)-$(SharedFxRID).zip"
```

Comment:
Can you also update publish settings since we won't be producing this anymore?

https://github.com/aspnet/Universe/blob/864ee50b4ce65136f2ee14b9e520015f555c1838/build/Publish.targets#L58-L74

Reply:
This too: https://github.com/aspnet/Universe/blob/91b852136bd28006f48c9323bccab1a2feabbf40/build/repo.props#L29-L31

Reply:
Updated. I'm updating my VS to test out the debugging experience.